pkgs/applications/graphics/freecad/default.nix

```diff
@@ -130,6 +130,8 @@ mkDerivation rec {
 
   postFixup = ''
     mv $out/share/doc $out
+    ln -s $out/bin/FreeCAD $out/bin/freecad
+    ln -s $out/bin/FreeCADCmd $out/bin/freecadcmd
```

Comment:
Not sure if we maybe just want to move them.

Lets wait a few days for the other maintainers.

Reply:
Sure, this change can wait. Thanks for the review as always. :smile:

Reply:
Symlink is the good thing to do here.
After all, if someone wrote a script expecting the original name, it will not break.